Frameworks/UIKit/UIButton.mm

```diff
@@ -71,16 +69,20 @@ @implementation UIButton {
     UIEdgeInsets _imageInsets;
     UIEdgeInsets _titleInsets;
 
-    // Proxies
-    StrongId<_UILabel_Proxy> _proxyLabel;
+    // Child elements
+    StrongId<UILabel> _titleLabel;
     StrongId<_UIImageView_Proxy> _proxyImageView;
 
     bool _isPressed;
+    UIButtonType _buttonType;
+
+    ComPtr<IInspectable> _inspectableAdjustsWhenDisabledBrush;
+    ComPtr<IInspectable> _inspectableAdjustsWhenHighlightedBrush;
 }
 
 /**
  @Status Caveat
- @Notes May not be fully implemented
+ @Notes Not all properties are supported.
```

Comment:
@tadam-msft, do we have a work item tracking the additiona of all missing UIButton xib2nib conversions?